internal/explorer/config_explorer.go

```diff
@@ -145,6 +160,17 @@ func extractOp(paths *high.Paths, oasLocation *config.OpenApiSpecLocation) (*hig
 	}
 }
 
+func extractParameters(paths *high.Paths, path string) ([]*high.Parameter, error) {
```

Comment:
Can we make this function name more explicit? Something like: `extractCommonParameters`